### PR TITLE
Fix/ndjson links

### PR DESCRIPTION
**Pull request by sfontana9**
## Description

The ndjson.org link became corrupted (the domain is pointing to some spam korean fake betting website), so I replaced it with the wikipedia paragraph reference of the newline delimited json.

## Pull Request Type

- [ ] Documentation content changes
- [x] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

**Comment by danibix95** [comment]
@sfontana9 thank you for the notice. Cloud you please also update the [CRUD Service documentation](https://github.com/mia-platform/crud-service/blob/main/docs/60_Usage_Best_Practices.md)? Otherwise, the next time it changes, it will overwrite this one upon its aggregation.

**Review by danibix95 — APPROVED** [review]

File: docs/runtime_suite/crud-service/60_usage_best_practices.md [hunk 1]
```diff
@@ -281,7 +281,7 @@ When the dataset of a collection has a high rate of **UPDATE/DELETE** operations
 The `GET /export` method exposed by each endpoint associated with a collection opens a data stream in `nd-json` format in the HTTP response. By using this method, the CRUD Service will open **only one cursor** to the MongoDB cluster, and the `ResultSet` will remain unaffected by concurrent **UPDATE/DELETE** operations.
 
 :::info
-[ndjson](http://ndjson.org/) is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
+[ndjson](https://en.wikipedia.org/wiki/JSON_streaming#Newline-delimited_JSON) is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
```
Comment (albertotessarotto) [inline docs/runtime_suite/crud-service/60_usage_best_practices.md:284]:
```suggestion
[ndjson] is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
```

File: docs/runtime_suite/crud-service/60_usage_best_practices.md [hunk 2]
```diff
@@ -281,7 +281,7 @@ When the dataset of a collection has a high rate of **UPDATE/DELETE** operations
 The `GET /export` method exposed by each endpoint associated with a collection opens a data stream in `nd-json` format in the HTTP response. By using this method, the CRUD Service will open **only one cursor** to the MongoDB cluster, and the `ResultSet` will remain unaffected by concurrent **UPDATE/DELETE** operations.
 
 :::info
-[ndjson](http://ndjson.org/) is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
+[ndjson] is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
```
Comment (albertotessarotto) [inline docs/runtime_suite/crud-service/60_usage_best_practices.md:284]:
```suggestion
ndjson is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
```

File: docs/runtime_suite/crud-service/60_usage_best_practices.md [hunk 3]
```diff
@@ -281,7 +281,7 @@ When the dataset of a collection has a high rate of **UPDATE/DELETE** operations
 The `GET /export` method exposed by each endpoint associated with a collection opens a data stream in `nd-json` format in the HTTP response. By using this method, the CRUD Service will open **only one cursor** to the MongoDB cluster, and the `ResultSet` will remain unaffected by concurrent **UPDATE/DELETE** operations.
 
 :::info
```
Comment (albertotessarotto) [inline docs/runtime_suite/crud-service/60_usage_best_practices.md:283]:
```suggestion
```

File: docs/runtime_suite/crud-service/60_usage_best_practices.md [hunk 4]
```diff
@@ -281,7 +281,7 @@ When the dataset of a collection has a high rate of **UPDATE/DELETE** operations
...
-[ndjson](http://ndjson.org/) is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
+ndjson is a format that ensures the streaming of data structures, where each record is processed individually and separated by a newline (`\n`) delimiter.
```
Comment (albertotessarotto) [inline docs/runtime_suite/crud-service/60_usage_best_practices.md:284]:
```suggestion
```

**Pull request merged**